.github/workflows/build.yaml

```diff
@@ -0,0 +1,12 @@
+on: [push, pull_request]
+name: Build
+jobs:
+    travis-ci:
+        name: Travis CI
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@master
+            - name: Travis CI
+              uses: travis-ci/actions@master
+              env:
+                  TRAVIS_TOKEN: ${{ secrets.TRAVIS_TOKEN }}
```

Comment:
@nicolas-grekas could you please set a secret for this repository with the symfony travis token?

Name: `TRAVIS_TOKEN`

Reply:
might be good

Reply:
@OskarStark You can relaunch ci with actual commit or you need commit a changed for launch ci ?

Reply:
Thanks Nicolas

Will push force in a minute